libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/permissions.spec.ts

```diff
@@ -229,17 +232,9 @@ describe("Dashboard", { tags: ["post-merge_integrated_tiger"] }, () => {
             Users.switchToUser(firstUser);
             Navigation.visit("dashboard/dashboard-tiger-permissions");
             new TopBar().enterSharing();
-            new ShareDialog()
-                .dialogExists(true)
-                .hasPermissionSet(firstGroup, "Can view & share")
-                .hasPermissionSet(firstUser, "Can view & share");
+            new ShareDialog().dialogExists(true).hasPermissionSet(firstGroup, "Can view & share");
```

Comment:
Can you please explain why we are removing this check for the user?
```
 .hasPermissionSet(firstUser, "Can view & share");
```
and later below the following checks
```
.openDropdownForUserOrGroup(firstUser)
                .isPermissionDisabled("Can edit & share")
                .isPermissionDisabled("Can view");
```
Are they no longer relevant or something?

Reply:
After the fix, all items will be disabled in the permission dropdown, so the dropdown will be disabled. That's why the old test will not work and replaced with this:
```javascript
new ShareDialog().isGranularPermissionButtonEnabled(firstUser, false);
```